components/properties/src/provider.rs

```diff
@@ -176,6 +202,125 @@ impl<'data, T: TrieValue> PropertyCodePointMapV1<'data, T> {
     }
 }
 
+// See ScriptWithExtensions for documentation of these functions
```

Comment:
Nit: Please move these functions to ScriptWithExtensionsBorrowed. There should be very little code in provider.rs files (I know we violate that sometimes but I really would like us to have this separation wherever possible)